recipes/vorbis/all/conanfile.py

```diff
@@ -34,7 +34,7 @@ def config_options(self):
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
```

Comment:
You'll also need to bump the required Conan version to 1.53.0: `required_conan_version = ">=1.53.0"